### ISSUE TITLE
Components, boards and accessories to add

### ISSUE THREAD

**Issue by bennuttall** [RPi-Distro/python-gpiozero#17]
Components:

- [x] Potentiometer
- [x] RGB LED
- [x] Ultrasonic sensor (HC - SR04)

Boards & accessories:

- [x] [Pi-Traffic](http://lowvoltagelabs.com/products/pi-traffic/)
- [x] [Pi stop](http://4tronix.co.uk/store/index.php?rt=product/product&product_id=390) (no standard pins so use generic `TrafficLights` class)
- [x] [Fish Dish](https://www.pi-supply.com/product/fish-dish-raspberry-pi-led-buzzer-board/)
- [x] [Pi Liter](http://shop.ciseco.co.uk/pi-liter-8-led-strip-for-the-raspberry-pi/)
- [x] [Ryanteck TrafficHat](https://ryanteck.uk/hats/1-traffichat-0635648607122.html)
- [x] [Ryanteck MCB](https://ryanteck.uk/add-ons/6-ryanteck-rpi-motor-controller-board-0635648607160.html)
- [x] [4tronix MCB from CamJam kit 3](http://camjam.me/?page_id=1035)

Longer-term additions:

- Energenie (see #69)
- LCD screens
- Servos (see [Grandpa Scarer](https://www.raspberrypi.org/learning/grandpa-scarer/) resource for RPi.GPIO examples)
- DHT (waiting on kernel module)
- Ph sensor?
- Quadrature rotation encoder motor ([e.g.](https://shop.pimoroni.com/products/optical-encoder-pair-kit-for-micro-metal-gearmotors-3-3v))
- Line follower

Not including things which already have decent libraries, like Pibrella and Explorer HAT.

**Comment by bennuttall** [comment]
Anything else?

**Comment by bennuttall** [comment]
I've half implemented the MCB - added a `Robot` class for two single-direction motor robots. RTK MCB uses two dual-direction motors.

**Comment by bennuttall** [comment]
Properly implemented Motor and Robot, including RTK MCB. Everything on the list is done now. Anything else to add?

**Comment by bennuttall** [comment]
Added suggestions from #56

**Comment by DexterInd** [comment]
Possible to put in a request for the Arduberry?

**Comment by DexterInd** [comment]
[Arduberry](http://www.dexterindustries.com/arduberry/) ?
[GrovePi] (http://www.dexterindustries.com/grovepi) ?

**Comment by bennuttall** [comment]
How do users currently use these products in software?

**Comment by DexterInd** [comment]
Hey Ben! 

The Arduberry is programmed in the Arduino language; it's a standalone Atmega328 connected to the Pi. Communication through either C or Python on the Pi.  

The GrovePi is mostly Python (though other languages are supported).  The Python library is a single python file [here on github](https://github.com/DexterInd/GrovePi/blob/master/Software/Python/grovepi.py).  

Did I answer your question?

**Comment by waveform80** [comment]
I've had a quick skim through the GrovePi's library that @DexterInd linked to; it looks like it's basically all I2C stuff so it's something we might want to add in future (given we've had a request for I2C based ADCs) but I doubt we can get it done for 1.0. Maybe stick it in a separate ticket for post-1.0 boards?

**Comment by bennuttall** [comment]
Yeah there are a few components listed here than aren't making the 1.0 release, so that's fine. Something to look at in future.

**Comment by DexterInd** [comment]
@bennuttall and @waveform80 , what's the best way for us to participate?  Would it make sense to do a pull request, or would that just muddle stuff up?

**Comment by lurch** [comment]
Something for the "Longer-term additions" list...
I dunno how well it'd mesh into the existing API, but it would be fantastic if gpiozero could also be used to provide a uniform interface to I2C IO-expanders, e.g. http://raspi.tv/2013/using-the-mcp23017-port-expander-with-wiringpi2-to-give-you-16-new-gpio-ports-part-3 and also used on boards like https://www.abelectronics.co.uk/products/17/Raspberry-Pi--Raspberry-Pi-2-Model-B/54/IO-Pi-Plus

**Comment by bennuttall** [comment]
Closing this - each component should have its own issue
